clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/SemiAutoClioIndex.scala

```diff
@@ -70,6 +73,7 @@ case object WgsUbamIndex
   override val name: String = "WgsUbam"
   override val commandName: String = "wgs-ubam"
   override val elasticsearchIndexName: String = "wgs-ubam"
+  override val defaults: Json = UbamMetadata(dataType = Option(DataType.WGS)).asJson
```

Comment:
The ticket is for crams & gvcfs, not ubams. data type isn't part of the key for ubams so it's OK for it to be missing on some upserts.

Reply:
(You'll need to add data type to those metadata classes)

Reply:
Sorry for scope creep, but could you also bundle https://broadinstitute.atlassian.net/browse/DSDEGP-2121 into this PR? Ideally we should only need to rebuild the cram & gvcf indexes once, so it'd be great if we started including data_type in the keys for crams & gvcfs as part of this.

Reply:
Oh wow I misunderstood the ticket. My bad!